### PR TITLE
LPS-148928 Remove hard pause from tests

### PR DESCRIPTION
**Pull request by susanchen3**
https://issues.liferay.com/browse/LPS-148928

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase [hunk 1]
```diff
@@ -145,10 +145,6 @@ definition {
 		}
 
 		task ("Assert the new button is displayed as an icon button") {
-			ScrollWebElementIntoView(locator1 = "ManagementBar#NEW_BUTTON_MOBILE");
-
-			Pause(locator1 = "3000");
-
```
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase:151]: It's not failing in [ManagementToolbar#ResponsiveModeNewButtonAsIconButton](https://testray.liferay.com/home/-/testray/cases/1627293472) so I removed it

**Comment by susanchen3** [comment]
ci:test:sf

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
Unrelated test failures. They all failed prior to running

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**